other/rainwatertrapping.py

```diff
@@ -0,0 +1,30 @@
+def maxTrappedWater(arr, n):
```

Comment:
```suggestion
def maxTrappedWater(arr):
    n = len(arr)
```
You should get the length inside, not pass it as a parameter